### PR TITLE
Base: Improve 3DFileViewer Icons and manpage

### PR DESCRIPTION
**Pull request by cubiclove**
New 32px and 16px application icons.
New 32px and optimized 16px filetype icons.
All four icons are now consistent, with a Lavender-blue Isometic Cube.

New vs Old

Contributes towards #21091

### CODE REVIEW

**Comment by cubiclove** [comment]
Hi @gmta. Would you be kind enough to proof read this please? I don't know much about 3D so have surely got a lot wrong.
The rotation axis especially confused me as each model seemed to act differently.

File: Base/usr/share/man/man1/Applications/3DFileViewer.md [hunk 1]
```diff
@@ -16,16 +16,46 @@ $ 3DFileViewer
...
+
+* **Enable Texture** - On by default, disable for a blank (gray) model.
+* **Wrap S or T** - Controls how a texture is applied horizontally (S) or vertically (T):
+    * **Repeat** - Tiles the texture repeatedly along the axis.
+    * **Mirrored** - Tiles and mirrors the texture along the axis.
+    * **Clamp** - Stretches or clamps the texture edges instead of repeating it.
+* **Scale** - Adjusts texture size (e.g. `0.5` reduces by half, `2x` doubles the size).
+* **Mag Filter** - Affects how model appears up-close:
+    * **Nearest** - Pixelated for a blocky look.
+    * **Linear** - Smoother by blending nearby texels (tiny pictures on a 3D object's surface).
```
Comment (gmta) [inline Base/usr/share/man/man1/Applications/3DFileViewer.md:49]: I would just reference the commonly known image scaling algorithms here; i.e. nearest neighbour and linear interpolation.

File: Base/usr/share/man/man1/Applications/3DFileViewer.md [hunk 2]
```diff
@@ -16,16 +16,46 @@ $ 3DFileViewer
...
+## Features
 
-By default 3D File Viewer will open with a rotating [Utah teapot](https://en.wikipedia.org/wiki/Utah_teapot), a standard 3D test model.
+By default, 3D File Viewer opens with a rotating [Utah teapot](https://en.wikipedia.org/wiki/Utah_teapot), a standard 3D test model. Open files via *File → Open* (`Ctrl+O`) or dragging and dropping files into the application. Examples files can be found in `Documents/3D Models`.
 
-Rotate the model by grabbing it with the cursor. Zoom-in and out with the mouse wheel.
+Rotate the model by grabbing it with the cursor, and zoom-in and out with the mouse wheel.
 
-The rotation axis and speed can be adjusted or turned off in the `View` menu.
+Settings persist whilst the application is open with different files, but does not persist between sessions.
```
Comment (gmta) [inline Base/usr/share/man/man1/Applications/3DFileViewer.md:25]: Typo: does -> do

**Comment by cubiclove** [comment]
Thanks for your feedback! I hope it's better now

File: Base/usr/share/man/man1/Applications/3DFileViewer.md [hunk 3]
```diff
@@ -7,25 +7,55 @@
...
+It currently supports opening the [Wavefront OBJ file format](https://en.wikipedia.org/wiki/Wavefront_.obj_file) (`.obj`).
 
-### Features
+## Features
 
-By default 3D File Viewer will open with a rotating [Utah teapot](https://en.wikipedia.org/wiki/Utah_teapot), a standard 3D test model.
+By default, 3D File Viewer opens with a rotating [Utah teapot](https://en.wikipedia.org/wiki/Utah_teapot), a standard 3D test model. Open files via *File → Open* (`Ctrl+O`) or dragging and dropping files into the application. Sample files can be found in `Documents/3D Models`.
 
-Rotate the model by grabbing it with the cursor. Zoom-in and out with the mouse wheel.
+Orbit around the model by grabbing it with the cursor, and zoom-in and out with the mouse wheel.
```
Comment (gmta) [inline Base/usr/share/man/man1/Applications/3DFileViewer.md:23]: Should be "zoom in" instead of "zoom-in"

**Pull request merged**